### ISSUE TITLE
Unable to install Blackarch aarch64 on Pinephone Pro

### ISSUE THREAD

**Issue by cmoa30** [BlackArch/blackarch-installer#98]
Hi,
I can't install Blackarch on my Pinephone Pro on fresh install of Arch aarch64.
When I use pacman -S blackarch, many packages are not downloading (webanalyze, skul, steghide, uefi-firmware-parser, recoverdm, rinetd, ssh-honeypot, radare2, zykeys, quicksand, recoverjpeg, wcvs, python2-roguehostapd, roguehostapd...) with the error "Maximum file size exceeded".
I tried to change from mirrors, no impact.
I change of Internet service provider, no impact.
I download manually these packages and push there on /var/cache/pacman/pkgs, pacman says they are corrupted and want delete them.
I downgrade gnugp to 2.2.41-2, remove gnugp directory, use gnugp --init, gnugp --populate arch archlinuxarm blackarch danctnix, update keyserver.ubuntu.com, no progress.
I search for an other answer other on Internet, I didn't find any.
Have you an idea for me for fix this problem ?
Thanks to you.

**Comment by ikstream** [comment]
Have you updated your system before trying this?
Did you use `-Syy` after changing the mirror?

please don’t run `pacman -S blackarch`. You’re trying to install a huge amount of packages at once.

**Comment by noraj** [comment]
> please don’t run `pacman -S blackarch`. You’re trying to install a huge amount of packages at once.

Yeah, please don't try to install 2800+ tools at once, it's guarantee to fail as there always are some broken packages. Only install the packages you need.

**Comment by cmoa30** [comment]
Thanks to yours responses, for explanation, I was able to install Blacharch almost completely by cloning the entire aarch64 repository with httrack locally, running "yay -U *.tar.gz" and "*.tar.xz", it took half a day to hunt for packages with insoluble dependencies but by doing so, I was able to install almost the Blackarch suite almost entirely. 
For reference to install Blackarch on a PinePhone Pro phone or other.
